html_css/css_basics.md

```diff
@@ -32,3 +32,4 @@ This section contains helpful links to other content. It isn't required, so cons
 
 * [Learn CSS Selectors Interactively Using CSS Diner](http://flukeout.github.io/)
 * [Smashing Magazine on Taming Advanced CSS Selectors](http://coding.smashingmagazine.com/2009/08/17/taming-advanced-css-selectors/)
+* [CSS Selectors Cheat Sheet](https://chl.li/cssSelectorCheatSheet)
```

Comment:
We try to stay away from downloads because they could potentially have malicious payloads/content. It is likely innocent but we don't want to risk it. If you can find a web link where it can be viewed and not downloaded it you can include that.